docs/modules/general/pages/implementation-details/webpack.adoc

```diff
@@ -0,0 +1,101 @@
+    '@ngx-translate/core': { requiredVersion: 'auto' },
+    primeng: { requiredVersion: 'auto', includeSecondaries: true },
+    rxjs: { requiredVersion: 'auto', includeSecondaries: true },
+    '@onecx/accelerator': { requiredVersion: 'auto', includeSecondaries: true },
+    '@onecx/angular-accelerator': { requiredVersion: 'auto', includeSecondaries: true },
+    '@onecx/angular-auth': { requiredVersion: 'auto', includeSecondaries: true },
+    '@onecx/angular-integration-interface': { requiredVersion: 'auto', includeSecondaries: true },
+    '@onecx/angular-remote-components': { requiredVersion: 'auto', includeSecondaries: true },
+    '@onecx/angular-testing': { requiredVersion: 'auto', includeSecondaries: true },
+    '@onecx/angular-webcomponents': { requiredVersion: 'auto', includeSecondaries: true },
+    '@onecx/integration-interface': { requiredVersion: 'auto', includeSecondaries: true },
+    '@onecx/keycloak-auth': { requiredVersion: 'auto', includeSecondaries: true },
+    '@onecx/ngrx-accelerator': { requiredVersion: 'auto', includeSecondaries: true },
+    '@onecx/portal-integration-angular': { requiredVersion: 'auto', includeSecondaries: true },
+    '@onecx/portal-layout-styles': { requiredVersion: 'auto', includeSecondaries: true }
+  }),
+
+  sharedMappings: []
+})
+
+module.exports = config
+
+const plugins = config.plugins.filter((plugin) => !(plugin instanceof ModifyEntryPlugin))
+
+module.exports = {
+  ...config,
+  plugins,
+  output: { uniqueName: 'example-ui', publicPath: 'auto' },
+  experiments: { ...config.experiments, topLevelAwait: true },
+  optimization: { runtimeChunk: false, splitChunks: false }
+}
+....
+
+The most important elements of Webpack configurations are:
+
+Exposed content::
+Each Microfrontend can expose multiple modules and Remote Components. Every separate module and Remote Component needs to be defined as a separate, unique entry representing a remote module. Each entry is a Key-Value Pair: 
++
+Key::::
+Name of the exposed module.
++
+Value::::
+File to be used for module bootstrap.
+
++
+In the Angular example, there are two remote modules defined:
+
+* **ExampleModule** - Microfrontend's remote module pointing to Microfrontend's main.ts file.
+* **ExampleComponent** - Remote Component's remote module pointing to component's main.ts file.
+
+IMPORTANT: https://webpack.js.org/concepts/module-federation/[Module Federation] considers any exposed content as a **remote module**. In the OneCX context, Microfrontends expose two types of content, being modules and Remote Components.
+
+Shared resources::
+// TODO: Link to OneCX package sharing
+Each Microfrontend can define packages that can be shared with others. Every Microfrontend can specify what packages and with what versions are allowed to be shared. In the OneCX context, it is important to ensure the correct package sharing configuration dependent on used packages and functionality used within the Microfrontend's exposed content.
```

Comment:
proposition: it is important to ensure the correct package sharing configuration dependent on the used packages used functionalities within the Microfrontend's exposed content.